www/install/php/Update-20.04.0-beta.1.php

```diff
@@ -48,21 +45,22 @@
     $configFilenames = [];
     while ($row = $result->fetch()) {
         $fileName = str_replace('.xml', '.json', $row['config_filename']);
+
+        // saving data for next engine module modifications
         $configFilenames[$row['config_filename']] = $fileName;
+
         $statement->bindValue(':value', $fileName, \PDO::PARAM_STR);
         $statement->bindValue(':id', $row['config_id'], \PDO::PARAM_INT);
-        $statement->execute();
+        if (false === $statement->execute()) {
+            throw new \PDOException(
```

Comment:
The CentreonDB class is already configured to generate an exception if an error occurs.